docs/configuration.rst

```diff
@@ -0,0 +1,129 @@
+.. _configuration file:
+
+*************
+Configuration
+*************
+
+Robocop provides a wide set of rules with good defaults and is designed to work well out-of-the-box.
+Nevertheless, it is natural that some projects may have different needs and requirements, which influence the way the code is written.
+Thus, Robocop allows to configure many rules to fit these needs.
+It's done using CLI options and such configuration can be stored and loaded from file, as described below.
+
+Configuring rules
+=================
+
+Rules are configurable. Severity of every rule message can be changed (see more :ref:`severity-threshold`) and also some of the rules have
+optional parameters.
+
+.. note::
+
+    To see all configurable rules and their parameters, run ``robocop --list-configurables`` or just ``robocop -lc``.
+
+For configuring rules you can use ``--configure`` or ``-c`` option followed by rule name (or ID), its parameter and the value delimited by colon (``:``)::
+
+    --configure <rule_name>:<param_name>:<value>
+
+For example::
+
+    robocop --configure line-too-long:line_length:140 .
+
+or more concise variant using rule ID instead of its name and short ``-c`` option::
+
+    robocop -c 0508:line_length:140 .
+
+which overwrites the value for the maximum line length (120 by default).
+
+Some rules accept comma-separated values like::
+
+    robocop -c todo-in-comment:markers:todo,changeme,refactor .
+
+which changes the markers that trigger the rule when the marker appears in the comment.
+
+If you need to provide a value with a space, wrap the whole configurable in quotes, like here::
+
+    robocop -c "todo-in-comment:markers:Remove me,Fix this!" .
+
+.. note::
+
+    Configuration parameter (``--configure`` / ``-c``) can also be used to configure reports. More about it here :ref:`configuring-reports`.
+
+Loading configuration from file
+===============================
+
+Robocop supports two formats of the configuration file: argument files and toml files. If argument file is not
```

Comment:
```suggestion
Robocop supports two formats of the configuration file: argument files and toml files. If configuration file is not
```

We are now support passing both argument and toml files though cli argument (--argumentfile or - - config), thats why we need to reword it